internal/controller/operator/configmap.go

```diff
@@ -374,6 +375,8 @@ func updatePlatformAuthIDP(_ common.SecondaryReconciler, _ context.Context, obse
 			"LDAP_CTX_POOL_PREFERREDSIZE"),
 		updatesValuesWhen(not(observedKeySet[*corev1.ConfigMap]("MASTER_PATH")),
 			"MASTER_PATH"),
+		updatesValuesWhen(not(observedKeySet[*corev1.ConfigMap]("LIBERTY_SAMESITE_COOKIE")),
+			"LIBERTY_SAMESITE_COOKIE"),
```

Comment:
We shouldn't need these lines because the value is already registered to always be updated on R327; you should remove this.